demos/CMakeLists.txt

```diff
@@ -154,6 +154,9 @@ macro(ie_add_sample)
             add_definitions(-DUSE_OPENCV)
         endif()
     endif()
+    if(WITHOUT_EXTENSIONS)
```

Comment:
There's no need for this to be an option. If there is an `IE::ie_cpu_extension` target, then use it. If there isn't, then don't.

Reply:
Fixed